### PR TITLE
[LuxLibraryHeader] [LuxMenuBar] Update to use font-size-base

### PR DESCRIPTION
**Pull request by christinach**
[LuxLibraryHeader] Update to use font-size-base
[LuxMenuBar] Update to use font-size-base

### CODE REVIEW

File: src/components/LuxLibraryHeader.vue [hunk 1]
```diff
@@ -186,7 +186,7 @@ export default {
   @media (min-width: 900px) {
     .full-name {
       display: block;
-      font-size: var(--font-size-large);
+      font-size: var(--font-size-base);
```
Comment (sandbergja) [inline src/components/LuxLibraryHeader.vue:189]: To me, having the app name the same size as the rest of the text looks odd

the header of the catalog, with the catalog looking a bit smaller than the menu items

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks, @christinach -- I wasn't sure about using the font-size-base for the app name, though.  To me, it should remain prominent.

File: src/components/LuxLibraryHeader.vue (hunk 1 above)
Comment (christinach) [reply src/components/LuxLibraryHeader.vue:189]: thanks Jane I didnt want to change this one.

**Review by sandbergja — APPROVED** [review]
Looks great, thanks @christinach ! 🐬

**Pull request merged**